1、解决监控界面图例压盖问题 2、解决设置界面ProcessInfo在亮主题下显示异常问题 3、增加监控方案管理功能

dk333 opened the pull request:
fix legend overlay​​s  title.

dk333 committed:
Update Monitor.vue

fix legend overlay​​s  title.

dk333 committed:
Update Settings.vue

fix  font color and bg color issue  in white theme

dk333 committed:
feat(Monitor): 添加 Kafka 监控方案保存和加载功能

feat(Monitor): 添加 Kafka 监控方案保存和加载功能

- 新增保存方案功能，可将当前选中的 Topics 和 Group 保存为方案
- 新增加载方案功能，可从已保存的方案中选择并应用
- 新增删除方案功能，可移除不再需要的方案
- 方案保存在本地存储中，支持持久化
- 优化了用户界面，增加了相关提示和确认操作

dk333 committed:
feat(i18n): 添加和优化巡检功能的国际化文案

feat(i18n): 添加和优化巡检功能的国际化文案

- 新增方案名称输入框、保存方案、删除方案等操作的文案
- 添加删除方案确认对话框的文案
- 补充必填字段提示、操作成功等提示信息

Bronya0 commented:
有一些建议：
1、国际化至少保证有中英，最好中英日
2、存储localstorage时我不确定wails项目之间是不是共享的，key尽量带个前缀标识，比如kafkaKing:key

dk333 committed:
Update en-US.js

调整监控界面监控方案的英语文案

dk333 committed:
Update Monitor.vue

给监控方案本地存储键增加前缀kafkaKing，避免冲突

dk333 committed:
fix(frontend): 修复主题更新逻辑

- 将 emitter.emit('update_theme', theme) 修改为 emitter.emit('update_theme', theme.value)
- 此修改确保在保存配置时正确发送更新后的主题值
